botorch/utils/objective.py

```diff
@@ -118,7 +120,7 @@ def apply_constraints(
     constraints: List[Callable[[Tensor], Tensor]],
     samples: Tensor,
     infeasible_cost: float,
-    eta: float = 1e-3,
+    eta: Union[List[float], float] = 1e-3,
```

Comment:
Why is it `List[float]` here and `Tensor` everywhere else?

Reply:
I forgot to update it, I will change it.

Reply:
Fixed.